HW5/TanchakYaroslav/loop_task.py

```diff
@@ -0,0 +1,45 @@
+# Task 1
+# list_numbers = [23, 45, 71, 12, 516, 3, 57, 10]
+#
+# for i in range(len(list_numbers)):
+#     list_numbers[i] = float(list_numbers[i])
+
+# Task 2
+
+# Version 1
+# fibonacci_list = [0, 1]
+# numb_fibonacci = int(input("Enter number for sequence of Fibonacci, please - "))
+#
+# for i in range(abs(numb_fibonacci)):
+#     next_fib_numb = fibonacci_list[-2] + fibonacci_list[-1]
+#     if next_fib_numb > numb_fibonacci:
+#         break
+#     fibonacci_list.append(next_fib_numb)
+#
+# print(fibonacci_list)
+
+#Version 2
+fibonacci_list = [0, 1]
+numb_fibonacci = abs(int(input("Enter number for sequence of Fibonacci, please - ")))
+
+while True:
+    next_fib_numb = fibonacci_list[-2] + fibonacci_list[-1]
+    if next_fib_numb > numb_fibonacci:
+        print("Next number for sequence of Fibonacci is higher than entered")
+        print("Sequence of Fibonacci is ready - ", fibonacci_list)
+        break
+    fibonacci_list.append(next_fib_numb)
+
+
+# Task 3
+
```

Comment:
good job
look at this version of code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)